go-apps/meep-ams/server/ams.go

```diff
@@ -671,30 +703,13 @@ func mec011AppTerminationPost(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
-func hasApplicationInfoChanged(appInfo1 *scc.ApplicationInfo, appInfo2 *scc.ApplicationInfo) bool {
-	if appInfo1 != nil || appInfo2 != nil {
-		if appInfo1 == nil && appInfo2 != nil {
-			return true
-		}
-		if appInfo1 != nil && appInfo2 == nil {
-			return true
-		}
-		if appInfo1.Id != appInfo2.Id {
-			return true
-		}
-		if appInfo1.Name != appInfo2.Name {
-			return true
-		}
-		if appInfo1.MepName != appInfo2.MepName {
-			return true
-		}
-		if appInfo1.Version != appInfo2.Version {
-			return true
-		}
-		if string(*appInfo1.Type_) != string(*appInfo2.Type_) {
-			return true
-		}
-		if string(*appInfo1.State) != string(*appInfo2.State) {
+func hasAppChanged(app1 *apps.Application, app2 *apps.Application) bool {
+	if app1 != nil || app2 != nil {
```

Comment:
Should consider using `go-test/deep` package to perform deep equality.
https://pkg.go.dev/github.com/go-test/deep#section-readme

Reply:
Fixed in PR #327